scripts/init-vlsi.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
-# exit script if any command fails 
+# exit script if any command fails
 set -e
 set -o pipefail
 
 # Initialize HAMMER and CAD-plugins
 git submodule update --init --recursive vlsi/hammer
-git submodule update --init --recursive vlsi/hammer-cad-plugins
```

Comment:
You need to modify exception for the cad-plugins to the `init-submodules-no-riscv-tools.sh` script, so base installations will not attempt to clone these private repos